### ISSUE TITLE
ts source compile many times after upgrade this plugin to 0.2.2

### ISSUE THREAD

**Issue by djx314** [ArpNetworking/sbt-typescript#44]
Hello, I use the sbt-typescript plugin in my playframework project.
When upgrade to version 0.2.2. It will compile all the typescript source after I change any other source or execute the `run` command thought I never change any ts source.
The out put like that

``` scala
[enuma] $ test:compile
Clean typescript cache
[info] TypeScript compiling on 54 source(s)
Clean typescript cache
[info] TypeScript compiling on 2 source(s)
Clean typescript cache
Clean typescript cache
[info] TypeScript compiling on 2 source(s)
Clean typescript cache
[success] Total time: 8 s, completed 2016-9-16 19:15:18
[enuma] $ run
Clean typescript cache
[info] TypeScript compiling on 54 source(s)
Clean typescript cache
[info] TypeScript compiling on 2 source(s)
Clean typescript cache
Clean typescript cache
[info] TypeScript compiling on 2 source(s)

--- (Running the application, auto-reloading is enabled) ---

[info] p.c.s.a.AkkaHttpServer - Listening for HTTP on /0:0:0:0:0:0:0:0:9000

(Server started, use Ctrl+D to stop and go back to the console...)
```

Any fix or workaround?

**Comment by wassertim** [comment]
Hey, it is the know issue and will be fixed soon. I will keep you updated.

**Comment by wassertim** [comment]
Hey this will be fixed when this PR is merged: https://github.com/ArpNetworking/sbt-typescript/pull/45

**Comment by djx314** [comment]
Oh, that's great!

**Comment by BrandonArp** [comment]
Fixed with #45.
